lib/que/worker.rb

```diff
@@ -11,8 +11,7 @@ module Que
   class Worker
     # Defines the time a worker will wait before checking Postgres for its next job
     DEFAULT_QUEUE = "default"
-    DEFAULT_WAKE_INTERVAL = 5
-    DEFAULT_LOCK_CURSOR_EXPIRY = 0 # seconds
+    DEFAULT_WAKE_INTERVAL = 5 # seconds
```

Comment:
The reason this became broken was because there's a weird two level default system here. I think we should keep one or the other, not both. 

Which place do you think is most appropriate to place the defaults? In the script entry point (bin/que) or in the worker?

Reply:
I've noticed that as well. Have consolidated it under `bin`, which is where the upstream que sets its defaults (i think that makes the most sense).

Reply:
I've noticed that as well. Have consolidated them under `bin`, as that's where upstream que sets its default values. Also moved the default `nil` value for `lock_window` and `lock_budget` into the same place.

Reply:
I did notice that too and originally thought of putting them all under `bin`. But after giving it some thought, the CLI might not be the only way people access `Que::Worker`, and we wouldn't want to introduce breaking changes there (still provide sensible defaults when people initialise a new worker directly). There's also a rare but not impossible case where the defaults we want when initialising a worker might differ from the defaults we want in the CLI.

Instead, in `bin` we reference the defaults set in `worker` - if we need to change worker-related defaults, we can directly change them in `worker` (TBF this was what it was originally).